deployment/terraform/README.md

```diff
@@ -1,100 +1,103 @@
+Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) on your box and make sure it is running `minikube start`.
 
-###Start
+#### Start
 From the root of the location to which haystack has been cloned:
 ```
 cd deployment/terraform
 ./apply-compose.sh -a install
 ```
-will install required third party software, start the minikube and install all haystack components in dev mode.
-###Verify
+this will install required third party software, start the minikube and install all haystack components in dev mode.
 
-#### How to access haystack-ui and grafana
-The script sets the 'haystack.local' domain to your minikube ip.
-use the below link access haystack at
-```
- https://haystack.local:32300
-```
-to check the health of the k8s cluster you can use the grafana link shared below
+#### Verify
+
+The script sets the 'haystack.local' domain point to your local minikube ip. Use the below link access to haystack UI -
 ```
- https://haystack.local:32300/grafana
+http://haystack.local:32300
 ```
 
-#### Verify haystack components
-Once scripts gets completed run the following command to see all the apps which are deployed
+Run the following command to see all the kubernetes pods which are deployed -
 ```
-kubectl get deployments --namespace=haystack-apps
+kubectl get pods --namespace=haystack-apps
 ```
 
-###Stop
+#### Stop
 From the root of the location to which haystack has been cloned:
 ```
 cd deployment/terraform
 ./apply-compose.sh -a uninstall
 ```
+
 this will uninstall all haystack components, but will leave minikube running. To bring down minikube:
 ```
 minikube stop
 ``` 
 Taking down minikube before running `./apply-compose.sh -a install` may prove helpful if minikube is returning errors
 during the install process.
 
-## How to deploy haystack on AWS?
-We support out of the box deployment in aws for haystack. The script uses teraform to create a kubernetes cluster and the rest of the dependent-infrastructure for haystack in aws
+
+
+## AWS Cluster
+We support out of the box deployment in aws for haystack. The script uses terraform to create a kubernetes cluster and the rest of the dependent-infrastructure for haystack in aws.
 Here are the list of components we install : 
 1. Kubernetes cluster - version 1.8 (using kops)
-2. Cassandra
+2. Apache Cassandra
 3. AWS Elastic Search
-4. Confluent-Platform Opensource
-5. Kubernetes Addons
-6. Haystack-Apps inside kubernetes
-
+4. Apache Kafka
+5. Kubernetes Addons (Traefik, Fluentd, ElasticSearch, Kibbana, Heapster, Influxdb, Graphite)
+6. Haystack apps inside kubernetes clusters
 
-###Pre-requisite 
-Haystack-AWS deployment requires you to create the following resources in aws and pass it as a parameter before you can trigger a deploy
+#### Pre-requisite 
+Haystack-AWS deployment requires you to create the following resources in aws and pass it as a parameter before you can trigger a deploy -
 1. AWS VPC
 2. AWS Subnet
 3. Hosted Zone Id
 4. S3 Bucket
 5. IAM User
```

Comment:
IAM use needs to be set and system properties in that box and not passed as a parameter